packages/web-config-server/src/apiV1/dataBuilders/helpers/calculateOperationForAnalytics.js

```diff
@@ -79,13 +80,47 @@ const combineBinaryIndicatorsToString = (analytics, config) => {
   return stringArray.length === 0 ? 'None' : stringArray.join(', ');
 };
 
+const combineTextIndicators = (analytics, config) => {
+  const { dataElements } = config;
+  const filteredAnalytics = analytics.filter(({ dataElement: de }) => dataElements.includes(de));
+  const stringArray = [];
+  filteredAnalytics.forEach(({ value }) => {
+    if (value) {
+      stringArray.push(value);
+    }
+  });
+  return stringArray.length === 0 ? 'None' : stringArray.join(', ');
+};
+
+const getMetaDataFromOrgUnit = async (_, config) => {
```

Comment:
@edmofro, it just occurred to me that this is also a model conflict.

Reply:
Yep! Thanks @Alastair-L

Reply:
@Alastair-L could you review my resolution of this https://github.com/beyondessential/tupaia/pull/1484/commits/279041d7cf12e1f4f82f3770343f68dc1cc40a53

BTW, I also made the other use of `calculateOperationForAnalytics` await, and made the function `async` to flag that it should always be awaited. It's not super safe to have it sometimes return a promise, and other times return the value directly - better to keep consistent so no one gets caught off guard.

Reply:
@edmofro, thanks for that!

Yes that makes sense, I didn't consider that 😬. The commit looks good, just added a few comments to consider.

Reply:
Sweet, thanks @Alastair-L